content/events/hacktoberfest.adoc

```diff
@@ -43,12 +43,21 @@ See the link:/participate/[Contribute and Participate] page for more information
 
 == Where to contribute?
 
-The Jenkins project is spread across several organizations on GitHub (jenkinsci, jenkins-x, jenkins-infra).
+The Jenkins project is spread across several organizations on GitHub (`jenkinsci`, `jenkins-infra`, `jenkins-zh`).
```

Comment:
Is Jenkins x no longer part of the Jenkins project?

Reply:
It is not, Jenkins X is a separate **friendly** project now, we have a reference to it in the Featured projects below. I am happy to keep it in the list if there is a preference, but IMHO it is not required (CC @MarckK )